components/supervisor/pkg/supervisor/services.go

```diff
@@ -753,6 +753,11 @@ func (c *ControlService) ExposePort(ctx context.Context, req *api.ExposePortRequ
 func (ss *ControlService) CreateSSHKeyPair(ctx context.Context, req *api.CreateSSHKeyPairRequest) (response *api.CreateSSHKeyPairResponse, err error) {
 	home := "/home/gitpod/"
 	userName := "gitpod"
+
+	ss.publicKey = req.PublicKey
+	ss.privateKey = req.PrivateKey
+
+	//TODO(aledbf): remove ssh key generation after we deploy the new version
```

Comment:
Do you mean remove CreateSSHKeyPair?

It will break VS Code Desktop and JB integration. They are working via HTTP tunnel and need to call it.